sdk/test/tutorials/createHoldings.test.ts

```diff
@@ -0,0 +1,554 @@
+              }
+            )
+          ]
+        }
+      ),
+
+    ]
+
+    client.api.transactionPortfolios.setHoldings(
+      portfolioObject.id.scope,
+      portfolioObject.id.code,
+      moment([2018, 1, 1, 0, 0, 0]).utc().format(),
+      holdingsAdjustments,
+    )
+    .then((res: {response: IncomingMessage, body: AdjustHolding}) => {
+
+      const transactions = [
+
+        Transactions.defineBuyRequest(
+          {
+            lusidInstrumentId: instrumentIDs[ 0 ],
+            units: 100.0,
+            price: 104.0,
+            currency,
+            transactionDate: moment([2018, 1, 5, 0, 0, 0]).utc()
+          }
+        ),
+
+        Transactions.defineBuyRequest(
+          {
+            lusidInstrumentId: instrumentIDs[ 2 ],
+            units: 100.0,
+            price: 103.0,
+            currency,
+            transactionDate: moment([2018, 1, 5, 0, 0, 0]).utc()
+          }
+        ),
+
+      ];
+
+      client.api.transactionPortfolios.upsertTransactions(
+        portfolioObject.id.scope,
+        portfolioObject.id.code,
+        transactions
+      )
+      .then((res: {response: IncomingMessage, body: UpsertPortfolioTransactionsResponse}) => {
+
+        client.api.transactionPortfolios.getHoldings(
+          portfolioObject.id.scope,
+          portfolioObject.id.code,
+          moment([2018, 1, 5, 0, 0, 0]).utc().format()
+        )
+        .then((res: {response: IncomingMessage, body: VersionedResourceListOfPortfolioHolding}) => {
+          resolve( res.body )
+        })
+        .catch((err: {response: IncomingMessage; body: LusidProblemDetails}) => reject(err))
+
+      } )
+      .catch((err) => mlog.error(err.response.statusCode, err.response.statusMessage, err.response.body.detail ) )
+
+    })
+    .catch((err: {response: IncomingMessage; body: LusidProblemDetails}) => reject(err))
+  });
+
+}
+
+//
+// Unit Test block
+//
+
+describe('Holdings', () => {
+
+  it('Should create a transaction portfolio', (done) => {
+    createTransactionPortfolio(uuidv4())
+    .then((res) => {
+
+      this.portfolioObject = res;
+
+      mlog.log( `Created portfolio ${res.displayName}` );
+
+      done()
+
+    })
+    .catch((err) => { mlog.error(err) } )
+  })
+
+  it('Should upsert instruments', (done) => {
+
+    upsertInstruments(
+      {
+        "BBG000FD8G46": {
+          Name: "HISCOX LTD"
+        },
+        "BBG000DW76R4": {
+          Name: "ITV PLC"
+        },
+        "BBG000PQKVN8": {
+          Name: "MONDI PLC"
+        },
+        "BBG000BDWPY0": {
+          Name: "NEXT PLC"
+        },
+        "BBG000BF46Y8": {
+          Name: "TESCO PLC"
+        },
+      }
+    )
+    .then((res) => {
+
+      this.instrumentsObject = res;
+      this.instrumentIDs = [];
+
+      Object.keys( res.values ).forEach( ( insertedInstrument ) => {
+
+        mlog.log( `Upserted instrument ${insertedInstrument} -> ${res.values[ insertedInstrument ].lusidInstrumentId}`)
```

Comment:
Should we also assert that the response contains the instruments that we expect?

Reply:
Or is that outside the scope of this test as it is just something we need to run the holding tests? If so, should the functions for creating these live in a utilities folder instead of inside this test file?